specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-01-23/keyvault.json

```diff
@@ -0,0 +1,1370 @@
+              "modelAsString": true
+            }
+          },
+          "description": "Permissions to certificates"
+        },
+        "storage": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "get",
+              "list",
+              "delete",
+              "set",
+              "update",
+              "regeneratekey",
+              "recover",
+              "purge",
+              "backup",
+              "restore",
+              "setsas",
+              "listsas",
+              "getsas",
+              "deletesas"
+            ],
+            "x-ms-enum": {
+              "name": "StoragePermissions",
+              "modelAsString": true
+            }
+          },
+          "description": "Permissions to storage accounts"
+        }
+      },
+      "description": "Permissions the identity has for keys, secrets, certificates and storage."
+    },
+    "VaultProperties": {
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The Azure Active Directory tenant ID that should be used for authenticating requests to the key vault."
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "SKU details"
+        },
+        "accessPolicies": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AccessPolicyEntry"
+          },
+          "description": "An array of 0 to 16 identities that have access to the key vault. All identities in the array must use the same tenant ID as the key vault's tenant ID. When `createMode` is set to `recover`, access policies are not required. Otherwise, access policies are required."
+        },
+        "vaultUri": {
+          "type": "string",
+          "description": "The URI of the vault for performing operations on keys and secrets."
+        },
+        "enabledForDeployment": {
+          "type": "boolean",
+          "description": "Property to specify whether Azure Virtual Machines are permitted to retrieve certificates stored as secrets from the key vault."
+        },
+        "enabledForDiskEncryption": {
+          "type": "boolean",
+          "description": "Property to specify whether Azure Disk Encryption is permitted to retrieve secrets from the vault and unwrap keys."
+        },
+        "enabledForTemplateDeployment": {
+          "type": "boolean",
+          "description": "Property to specify whether Azure Resource Manager is permitted to retrieve secrets from the key vault."
+        },
+        "enableSoftDelete": {
+          "type": "boolean",
+          "description": "Property to specify whether the 'soft delete' functionality is enabled for this key vault. If it's not set to any value(true or false) when creating new key vault, it will be set to true by default. Once it's been set to true value, it can NOT be reverted to false."
```

Comment:
Please specify the `default` value as True. It implies exactly what you are explaining in the description.